bliss/models/location_encoder.py

```diff
@@ -394,19 +386,15 @@ def _get_normal_samples(mean, sd, tile_is_on_array):
     def _get_hidden_indices(self):
         """Setup the indices corresponding to entries in h, cached since same for all h."""
 
-        # initialize matrices containing the indices for each variational param.
+        # initialize matrices containing the indices for each distributional param.
```

Comment:
I'm gradually changing "variational" to "distribution" in this file. "Distributional" isn't that descriptive, but "variational" is incorrect. The variational parameters are the weights of the encoders networks.